python/cuml/test/test_kmeans.py

```diff
@@ -104,10 +110,10 @@ def test_kmeans_sequential_plus_plus_init(nrows, ncols, nclusters,
 
 
 @pytest.mark.parametrize('nrows', [1000, 10000])
-@pytest.mark.parametrize('ncols', [10, 50])
+@pytest.mark.parametrize('ncols', [25])
 @pytest.mark.parametrize('nclusters', [2, 5])
 @pytest.mark.parametrize('cluster_std', [1.0, 0.1, 0.01])
-@pytest.mark.parametrize('random_state', [i for i in range(25)])
+@pytest.mark.parametrize('random_state', [i for i in range(20)])
```

Comment:
Hmm... this runs 20 times to test 20 random seeds? Maybe there is an interesting backstory why this is necessary.. any ideas on that @cjnolet ?
My gut would be to greatly reduce the number of seeds tested

Reply:
I can reduce it if it is not important. I reduce some of the random seed iterations already.